src/gui/torrentcontentmodel.cpp

```diff
@@ -355,3 +459,23 @@ void TorrentContentModel::selectNone()
         m_rootItem->child(i)->setPriority(prio::IGNORED);
     emit dataChanged(index(0, 0), index(rowCount(), columnCount()));
 }
+
+bool TorrentContentModel::useSystemFileIcons()
+{
+    return SettingsStorage::instance()->loadValue(useSystemFileIconsSettingKey, false).toBool();
+}
+
+void TorrentContentModel::useSystemFileIcons(bool use)
+{
+    m_useSystemFileIcons = use;
+    SettingsStorage::instance()->storeValue(useSystemFileIconsSettingKey, use);
+}
+
+bool TorrentContentModel::getUseSystemFileIcons()
```

Comment:
CachedSettingsValue will help you avoid having this function and one extra variable.